frontend/src/assets/styles/styles.scss

```diff
@@ -1,9 +1,22 @@
+@use 'sass:map';
+@use '@bcgov-nr/nr-fsa-theme/design-tokens/type-family.scss' as typeFamily;
+
 // Import all of Bootstrap's CSS
 @import '~bootstrap/scss/bootstrap';
 
+@import '../../../node_modules/primeicons/primeicons.css';
+
 @import 'base';
 
 // Components
 @import './components/form-elements.scss';
 
 @import '@bcgov-nr/nr-fsa-theme/style-sheets/primevue-components-overrides.scss';
+
+:root {
+    --font-family: map.get(typeFamily.$type-family, 'sans');

```

Comment:
My css knowledge is limited. My understanding is here is to set "--font-family" **css variable** not the **css style**.
Curious that is this variable being used anywhere? Since at the line 21 below this line in the "body {}" it does not use this variable. So my impression on this line is it is only declare this variable but not being used. Or maybe I am wrong, could you explain a bit? Thank you.